prometheus/prometheus.go

```diff
@@ -2,17 +2,22 @@ package prometheus
 
 import (
 	"context"
+	"errors"
+	"fmt"
 	"time"
 
 	"github.com/99designs/gqlgen/graphql"
 	prometheusclient "github.com/prometheus/client_golang/prometheus"
+	"github.com/vektah/gqlparser/gqlerror"
 )
 
 const (
 	existStatusFailure = "failure"
 	exitStatusSuccess  = "success"
 )
 
+const errCodeExt = "error_code"
```

Comment:
join const declaration with 2 already existing above?

Reply:
I prefer separate independent const/var groups

Reply:
but they are connected. 3 of them for a single Prometheus metric.